### PR TITLE
Add h erg blocker from TDC

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: data/hERG_blockers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: hERG
+description: "Human ether-\xE0-go-go related gene (hERG) is crucial for the coordination\
```
Comment (kjappelbaum) [inline data/hERG_blockers/meta.yaml:2]: shall we use unicode instead of -\xE0-?

**Review by phalem — COMMENTED** [review]
Replace \xE0 with a

**Review by phalem — COMMENTED** [review]
Add benchmark field

File: data/herg_blockers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+name: herg_blockers
+description: "Human ether-à-go-go related gene (hERG) is crucial for the coordination\nof the heart's beating. Thus, if a drug blocks the hERG, it could\
+    \ lead to severe\nadverse effects. Therefore, reliable prediction of hERG liability in the early\nstages of drug design is quite important to reduce\
+    \ the risk of cardiotoxicity\nrelated attritions in the later development stages."
```
Comment (kjappelbaum) [inline data/herg_blockers/meta.yaml:5]: same question here about the `\`

File: data/herg_blockers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          - noun: hERG blocker
+          - noun: hERG active compound
+          - noun: hERG active compound (<10uM)
+          - noun: human ether-à-go-go related gene (hERG) blocker
+          - verb: blocks hERG
+          - verb: blocks human ether-à-go-go related gene (hERG)
+          - verb: active against hERG (<10uM)
+          - verb: active against human ether-à-go-go related gene (hERG)
+      uris:
+          - http://purl.obolibrary.org/obo/MI_2136
```
Comment (kjappelbaum) [inline data/herg_blockers/meta.yaml:21]: this is only referring to the binder - do we want to keep it?

Comment (kjappelbaum) [reply data/herg_blockers/meta.yaml:21]: otherwise, lgtm!

Comment (MicPie) [reply data/herg_blockers/meta.yaml:21]: If you think this doesn't make sense, we can remove it.

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**